### PR TITLE
Update couchbase chart version to latest

### PR DESCRIPTION
**Pull request by arush-sal**
## Change Overview

Updated the Couchbase app helm chart version to the latest from 0.1.2

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [x] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [ ] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #XXX

## Test Plan

* $ kind create cluster
* $ make install-minio

```
make[1]: Entering directory '/root/go/src/github.com/kanisterio/kanister'
running CMD in the containerized build environment
Deploying minio...
"stable" has been added to your repositories
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "stable" chart repository
Update Complete. ⎈ Happy Helming!⎈ 
namespace/minio created
NAME: minio
LAST DEPLOYED: Tue Sep  8 15:42:35 2020
NAMESPACE: minio
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
Minio can be accessed via port 9000 on the following DNS name from within your cluster:
minio.minio.svc.cluster.local

To access Minio from localhost, run the below commands:

  1. export POD_NAME=$(kubectl get pods --namespace minio -l "release=minio" -o jsonpath="{.items[0].metadata.name}")

  2. kubectl port-forward $POD_NAME 9000 --namespace minio

Read more about port forwarding here: http://kubernetes.io/docs/user-guide/kubectl/kubectl_port-forward/

You can now access Minio server on http://localhost:9000. Follow the below steps to connect to Minio server with mc client:

  1. Download the Minio mc client - https://docs.minio.io/docs/minio-client-quickstart-guide

  2. mc config host add minio-local http://localhost:9000 AKIAIOSFODNN7EXAMPLE wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY S3v4

  3. mc ls minio-local

Alternately, you can use your browser or the Minio SDK to access the server - https://docs.minio.io/categories/17

Use following creds to access MinIO
AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
AWS_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
AWS_REGION=us-west-2
LOCATION_ENDPOINT=http://minio.minio.svc.cluster.local:9000

minio deployment successful!
make[1]: Leaving directory '/root/go/src/github.com/kanisterio/kanister'

```

* $ make integration-test short

https://gist.github.com/arush-sal/c71fdf0f2cf31471c1936c5166c35d43

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [x] :green_heart: E2E

### CODE REVIEW

File: pkg/app/couchbase.go [hunk 1]
```diff
@@ -49,15 +49,16 @@ type CouchbaseDB struct {
...
+// Last tested woking version "0.1.2"
 func NewCouchbaseDB(name string) App {
 	return &CouchbaseDB{
 		name: name,
 		operatorChart: helm.ChartInfo{
 			Release:  appendRandString(fmt.Sprintf("%s-operator", name)),
 			RepoName: helm.CouchbaseRepoName,
 			RepoURL:  helm.CouchbaseRepoURL,
-			Chart:    "couchbase-operator",
 			Version:  "0.1.2",
```
Comment (viveksinghggits) [inline pkg/app/couchbase.go:60]: do we want to remove this version from here?

File: pkg/app/couchbase.go [hunk 2]
```diff
@@ -106,7 +107,10 @@ func (cb *CouchbaseDB) Install(ctx context.Context, ns string) error {
 
 	// Install cb cluster
 	err = cli.Install(ctx, fmt.Sprintf("%s/%s", cb.operatorChart.RepoName, cb.clusterChart.Chart), cb.clusterChart.Version, cb.clusterChart.Release, cb.namespace, cb.clusterChart.Values)
-	return errors.Wrapf(err, "Failed to install helm chart. app=%s chart=%s release=%s", cb.name, cb.clusterChart.Chart, cb.clusterChart.Release)
+	if err != nil {
```
Comment (viveksinghggits) [inline pkg/app/couchbase.go:110]: if err is nil, errors.Wrapf would return nil automatically right. Or thats not happening.

File: pkg/app/couchbase.go (hunk 1 above)
Comment (ankitjain235) [reply pkg/app/couchbase.go:60]: Installation of the latest version of couchbase helm chart itself is failing. Removing version will cause the test to fail.

File: pkg/app/couchbase.go (hunk 2 above)
Comment (ankitjain235) [reply pkg/app/couchbase.go:110]: Okay, wasn't aware of that. Will revert this.

File: pkg/app/couchbase.go [hunk 3]
```diff
@@ -40,34 +40,29 @@ const cbReadyTimeout = 5 * time.Minute
...
-	cli           kubernetes.Interface
-	operatorChart helm.ChartInfo
-	clusterChart  helm.ChartInfo
+	name      string
+	namespace string
+	username  string
+	password  string
+	cli       kubernetes.Interface
+	chart     helm.ChartInfo
+	// clusterChart  helm.ChartInfo
```
Comment (viveksinghggits) [inline pkg/app/couchbase.go:49]: should we remove this?

**Review by viveksinghggits — CHANGES_REQUESTED** [review]

File: pkg/app/couchbase.go (hunk 3 above)
Comment (ankitjain235) [reply pkg/app/couchbase.go:49]: Yes, removing this now.

**Review by viveksinghggits — APPROVED** [review]

**Pull request merged**